admin/backend/utils/view-helpers.js

```diff
@@ -4,18 +4,29 @@ const lodash = require('lodash')
 class ViewHelpers {
   constructor({ admin }) {
     this._admin = admin
-
     this.paginate = paginate
     this._ = lodash
     this.branding = this._admin.options.branding
   }
 
+  getObjectKeyWithValue(obj, key) {
```

Comment:
not sure but maybe you can use url.parse(....): https://nodejs.org/api/url.html ?

Reply:
I dont understand, but i think it works fine. We can have nested objects so recursion is needed

Reply:
In this case I cant use url.parse